### PR TITLE
Fixes #19695

### LINKED ISSUE #19695: You pull back the tab of Nuka Cola Quantum with a satisfying pop.
(WEB REPORT BY: mask2697 REMOTE: 172.93.109.202:7777)
# Revision
80f35b7d06c1dab7bb9ff18777c95874b968addd

# Description
Its a fucking bottle, BOTTLES DONT HAVE TABS
# Steps to Reproduce

Get Nuka Cola Quantum, open with satisfying pop
# What you Expected

To remove the cap with a satisfying pop
# What Actually Happened

Remove tab.

### PR DESCRIPTION
**Pull request by etherware-novice**
## What this does
Fixes issue #19695 

## Why it's good
simple pr to get back into coding, also improves immersion and stuff

## Changelog

:cl:
 * spellcheck: Changes nuka cola's tab message to say cap

### CODE REVIEW

File: code/modules/reagents/reagent_containers/food/drinks.dm [hunk 1]
```diff
@@ -718,7 +720,7 @@
 		qdel(src)
 
 /obj/item/weapon/reagent_containers/food/drinks/soda_cans/proc/pop_open(var/mob/user)
-	to_chat(user, "You pull back the tab of \the [src] with a satisfying pop.")
+	to_chat(user, "[tabself] of \the [src] with a satisfying pop.")
```
Comment (jwhitak) [inline code/modules/reagents/reagent_containers/food/drinks.dm]: This "of" is unneeded, since the tabself variable has the ofs already.

Comment (etherware-novice) [reply code/modules/reagents/reagent_containers/food/drinks.dm]: was pinged about it on the discord but thx

**Comment by Hinaichigo** [comment]
Why not a var like `var/captab = "tab"` to make it simpler?

**Comment by Inorien** [comment]
> You pull back the cap of the bottle of water

"You pop the cap off the ..." ?

**Comment by SonixApache** [comment]
for future ref: write a descriptive title and comment, put 'fixes x' in there for good measure
what you wrote in your changelog would've been a good title
also changelogs should be past tense and descriptive of the issue resolved if it wasn't obvious - in this case, i'd go with something along the lines of "You now pop off the cap instead of pulling the tab off a nuka cola bottle"

File: code/modules/reagents/reagent_containers/food/drinks.dm [hunk 2]
```diff
@@ -819,6 +821,8 @@
 	name = "Nuka Cola"
 	desc = "Cool, refreshing, Nuka Cola."
 	icon_state = "nuka"
+	tabself = "You pull back the cap on"
```
Comment (Kurfursten) [inline code/modules/reagents/reagent_containers/food/drinks.dm]: Ino is correct, this language doesn't sound natural. Caps are unscrewed or popped off, not pulled back like tabs.

Comment (etherware-novice) [reply code/modules/reagents/reagent_containers/food/drinks.dm]: would "pop off the cap" work?

Comment (Kurfursten) [reply code/modules/reagents/reagent_containers/food/drinks.dm]: "Pop the cap off" would do

**Pull request merged**